otx/algorithms/segmentation/configs/configuration.yaml

```diff
@@ -70,7 +70,7 @@ learning_parameters:
     affects_outcome_of: TRAINING
```

Comment:
Please revert all base config changes.

Reply:
It is better to extend early stopping patience  for 10 epochs and turn it on a bit later. Now, it exits from training to early sometimes